### ISSUE TITLE
ZTP error on CVP-2020.2.3

### ISSUE THREAD

**Issue by 4ever7** [aristanetworks/ansible-cvp#279]
## Issue Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation Report
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Other (please describe):

## Summary

underneath the host section of the dhcpd.conf file, the "hardware ethernet aa:bb:cc:dd:ee:ff;" seems to cause errors on the EOS platform ( %ZTP-4-DHCPv4_QUERY_FAIL: Failed to get a valid DHCPv4 response).  Once the format changes to "option dhcp-client-identifier aa:bb:cc:dd:ee:ff", the EOS device accepts the dhcp response from server

A CVP instance (2020.2.3) is acting as my dhcp server

according to the online public documentation, the syntax seems to of changed in 2020.2.0:
https://www.arista.com/en/cg-cv/cv-dhcp-service-for-zero-touch-provisioning-ztp-setup

## Role or Module Name

__Module name:__ `module name`

`arista.cvp` collection and Python libraries version

```shell
# Ansible version
$ ansible --version

# list of python packages if required.
$ pip freeze

```

## OS / Environment

__Cloudvision version__

__EOS Version__

__OS running Ansible__

## Steps to reproduce

```yaml
---
```

### Expected results

```shell
$
```

### Actual results

```shell
$
```

**Comment by titom73** [comment]
I am trying to reproduce issue in lab using following versions:

- Cloudvision: `2020.2.1`
- vEOS: `4.24.0.F`

With the following `dhcpd.conf` file, I get following result on EOS side:

- DHCP configuration

```conf
# Ansible managed - cv_ztp
# Subnet of ZTP interface
subnet 10.73.255.0 netmask 255.255.255.0 {
    range 10.73.255.200 10.73.255.250;
    option routers 10.73.255.2;
    option domain-name-servers 10.73.255.2;
    max-lease-time 300;
}

# Per host definition
host DC1-SPINE1 {
    option host-name "DC1-SPINE1";
    # hardware ethernet 0c:1d:c0:1d:62:01;
    option dhcp-client-identifier 0c:1d:c0:1d:62:01;
    fixed-address 10.73.255.101;
    option bootfile-name "http://10.73.255.1/ztp/bootstrap";
    option routers 10.73.255.2;
    option domain-name-servers 10.73.255.2;
}
```

- Process status

```shell
[root@cvp ~]# service dhcpd status
Redirecting to /bin/systemctl status dhcpd.service
● dhcpd.service - DHCPv4 Server Daemon
   Loaded: loaded (/usr/lib/systemd/system/dhcpd.service; enabled; vendor preset: disabled)
   Active: active (running) since Tue 2020-11-17 14:03:18 UTC; 17min ago
     Docs: man:dhcpd(8)
           man:dhcpd.conf(5)
 Main PID: 1189 (dhcpd)
   Status: "Dispatching packets..."
    Tasks: 1
   Memory: 5.6M
   CGroup: /system.slice/dhcpd.service
           └─1189 /usr/sbin/dhcpd -f -cf /etc/dhcp/dhcpd.conf -user dhcpd -group dhcpd --no-pid

Nov 17 14:18:09 cvp.lab.as73.inetsix.net dhcpd[1189]: DHCPREQUEST for 10.73.255.202 from 0c:1d:c0:1d:62:21 via ens224
Nov 17 14:18:09 cvp.lab.as73.inetsix.net dhcpd[1189]: DHCPACK on 10.73.255.202 to 0c:1d:c0:1d:62:21 via ens224
```

- Network traces

```
14:08:54.256081 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from 0c:1d:c0:1d:62:21 (oui Unknown), length 300
14:08:55.271226 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from 0c:1d:c0:1d:62:21 (oui Unknown), length 300
14:09:24.669661 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from 0c:1d:c0:1d:62:21 (oui Unknown), length 300
14:09:55.898331 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from 0c:1d:c0:1d:62:21 (oui Unknown), length 300
```

- EOS Cli in ZTP mode:

```
Nov 17 14:14:17 localhost ZeroTouch: %ZTP-4-DHCPv4_QUERY_FAIL: Failed to get a valid DHCPv4 response
Nov 17 14:14:17 localhost ZeroTouch: %ZTP-6-RETRY: Retrying Zero Touch Provisioning from the beginning (attempt 11)
Nov 17 14:14:33 localhost ZeroTouch: %ZTP-6-DHCPv4_QUERY: Sending DHCPv4 request on  [ Ethernet1, Ethernet2, Ethernet3, Ethernet4, Ethernet
5, Ethernet6, Ethernet7, Ethernet8, Management1 ]
```

Could you share which version of EOS you are using in your setup ?

**Comment by titom73** [comment]
Documentation gives following reason:

- `option dhcp-client-identifier`: System MAC address of the device [i.e., MAC visible in `show version` output]
- `hardware ethernet`: MAC address of the interface sending the DHCP request (e.g., Management1)

PR in progress
